### PR TITLE
community: add afreen as the dashboard lead

### PR DESCRIPTION
**Pull request by nizamial09**

### CODE REVIEW

File: src/en/community/team/index.html [hunk 1]
```diff
@@ -58,7 +58,7 @@ <h3 class="h3">Component Leads</h3>
               <td>RADOS</td>
             </tr>
             <tr>
-              <td><a class="a" href="#">Nizamudeen A</a></td>
+              <td><a class="a" href="#">Afreen Misbah</a></td>
```
Comment (epuertat) [inline src/en/community/team/index.html:61]: Hey Nizam, just talked about this with @afreen23. I strongly suggest that both of you add a profile picture and small bio to the bottom section :D 

BTW, could you plz fix my bio section below?:
```diff
- Ernesto is the Ceph Dashboard component lead.
+ Ernesto was the Ceph Dashboard component lead from 2020 to 2022.
```

Comment (nizamial09) [reply src/en/community/team/index.html:61]: sorry @epuertat i forgot to come and check this PR. Yup, I'll fix the bio there and update this PR.

btw, the bottom section was reserved for maintainers isn't it?

**Comment by jdurgin** [comment]
do we need any updates here, or can we merge and make another pr for separate changes?

File: src/en/community/team/index.html (hunk 1 above)
Comment (nizamial09) [reply src/en/community/team/index.html:61]: @afreen23 let's add a bio section at the bottom. I'll add one for myself too. The photo might be optional there but its good to leave something there so people can easily contact us.

For now we can merge this PR and take it in separately.

**Comment by nizamial09** [comment]
> or can we merge and make another pr for separate changes?

Yup, we can merge this and take the other changes separately. @afreen23 could you approve and merge this in? thanks

File: src/en/community/team/index.html [hunk 2]
```diff
@@ -216,11 +216,11 @@ <h3 class="h3" id="neha-ojha">Neha Ojha</h3>
...
-              where he first came to know about Ceph, for a Cloud Video Storage project back in 2015. After that stimulating experience, he
-              joined Red Hat in 2018 and has since contributed to the Ceph Dashboard project, trying to apply his expertise as a Ceph user.
-              Ernesto holds a master’s degree in Telecommunications Engineering from Universidad Politénica de Madrid and currently lives in
-              that very city, famous for its fried calamari sandwich.
+              Ernesto was the Ceph Dashboard component lead from 2020 to 2022. He previously worked at Telefonica R&mp;D, Alcatel-Lucent,
+              Bell Labs, and Nokia, where he first came to know about Ceph, for a Cloud Video Storage project back in 2015. After that
+              stimulating experience, he joined Red Hat in 2018 and has since contributed to the Ceph Dashboard project, trying to apply his
+              expertise as a Ceph user. Ernesto holds a master’s degree in Telecommunications Engineering from Universidad Politénica de
+              Madrid and currently lives in that very city, famous for its fried calamari sandwich.
```
Comment (nizamial09) [inline src/en/community/team/index.html:223]: these changes looks like lint

**Review by afreen23 — APPROVED** [review]

**Pull request merged**